x-pack/plugins/cases/public/components/case_action_bar/index.tsx

```diff
@@ -172,15 +172,9 @@ const CaseActionBarComponent: React.FC<CaseActionBarProps> = ({
                     </EuiButtonEmpty>
                   </span>
                 </EuiFlexItem>
-                {/* currently just delete should we limit it to that? */}
-                {permissions.all && (
-                  <EuiFlexItem grow={false} data-test-subj="case-view-actions">
-                    <Actions
-                      caseData={caseData}
-                      currentExternalIncident={currentExternalIncident}
-                    />
-                  </EuiFlexItem>
-                )}
+                <EuiFlexItem grow={false} data-test-subj="case-view-actions">
```

Comment:
@cnasikas This will cause some whitespace at the end. If we keep with disabling the entire flex item them it fits better. It seems like a better idea to do the permissions check within the component though, what do you think?

![image](https://user-images.githubusercontent.com/56361221/178070275-7595a9ee-a5bd-459d-958e-7fd1554fe426.png)

Reply:
We talked about this offline, we're going allow the link to be shown and only remove the delete case button if the user doesn't have deletion permissions.